### PR TITLE
Examples deploying LLM apps to Online Endpoints

### PR DESCRIPTION
**Pull request by varndorfer**
# Description

Two examples using common LLM application frameworks (Semantic Kernel and Langchain) and deploying them to an AzureML Online Endpoint.

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/python/README.md [hunk 1]
```diff
@@ -68,10 +72,65 @@ Test Status is for branch - **_main_**
...
+|featurestore_sample|notebooks|[1. Develop a feature set and register with managed feature store](featurestore_sample/notebooks/sdk_and_cli/1. Develop a feature set and register with managed feature store.ipynb)|*no description*|[![1. Develop a feature set and register with managed feature store](https://github.com/Azure/azureml-examples/actions/workflows/sdk-featurestore_sample-notebooks-sdk_and_cli-1. Develop a feature set and register with managed feature store.yml/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/sdk-featurestore_sample-notebooks-sdk_and_cli-1. Develop a feature set and register with managed feature store.yml)|
```
Comment (wayliums) [inline sdk/python/README.md:75]: how come your PR has feature store notebook changes

File: sdk/python/README.md [hunk 2]
```diff
@@ -50,11 +49,16 @@ Test Status is for branch - **_main_**
...
 |endpoints|online|[kubernetes-online-endpoints-simple-deployment](endpoints/online/kubernetes/kubernetes-online-endpoints-simple-deployment.ipynb)|Use an online endpoint to deploy your model, so you don't have to create and manage the underlying infrastructure|[![kubernetes-online-endpoints-simple-deployment](https://github.com/Azure/azureml-examples/actions/workflows/sdk-endpoints-online-kubernetes-kubernetes-online-endpoints-simple-deployment.yml/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/sdk-endpoints-online-kubernetes-kubernetes-online-endpoints-simple-deployment.yml)|
+|endpoints|online|[1_langchain_basic_deploy](endpoints/online/llm/langchain/1_langchain_basic_deploy.ipynb)|*no description*|[![1_langchain_basic_deploy](https://github.com/Azure/azureml-examples/actions/workflows/sdk-endpoints-online-llm-langchain-1_langchain_basic_deploy.yml/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/sdk-endpoints-online-llm-langchain-1_langchain_basic_deploy.yml)|
```
Comment (wayliums) [inline sdk/python/README.md:60]: can you add descriptions?

File: sdk/python/endpoints/online/llm/langchain/deployments/env.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: plugin-demo-env
```
Comment (wayliums) [inline sdk/python/endpoints/online/llm/langchain/deployments/env.yml:1]: let's remove any naming related to plugin

File: sdk/python/README.md (hunk 1 above)
Comment (varndorfer) [reply sdk/python/README.md:75]: This file is generated by running the readme.py file ([per the contributing guide instructions](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md#set-up-and-pre-pr)). I guess whoever wrote those notebooks didn't run readme.py before completing their PR

File: sdk/python/README.md (hunk 2 above)
Comment (varndorfer) [reply sdk/python/README.md]: added

File: sdk/python/endpoints/online/llm/langchain/deployments/env.yml (hunk 3 above)
Comment (varndorfer) [reply sdk/python/endpoints/online/llm/langchain/deployments/env.yml]: changed to llm-demo-env

**Review by wayliums — APPROVED** [review]

**Pull request merged**